pursuit.cabal

```diff
@@ -85,7 +85,7 @@ library
                  , directory                     >= 1.1        && < 1.3
                  , warp                          >= 3.0        && < 4
                  , data-default
-                 , aeson                         >= 0.6        && < 0.10
+                 , aeson                         >= 0.6        && < 0.12
```

Comment:
I mean I don't want to worry about everything breaking in case someone uses `cabal-install` instead of `stack` for whatever reason, and the dependency solver selects an older version of `aeson`.